modules/api/pkg/api/v2/types.go

```diff
@@ -2060,6 +2060,9 @@ type GlobalSettings struct {
 
 	EnableOIDCKubeconfig bool `json:"enableOIDCKubeconfig"` //nolint:tagliatelle
 
+	// EnableClusterBackup enables the Cluster Backup feature in the dashboard.
+	EnableClusterBackups *bool `json:"enableClusterBackup,omitempty"`
```

Comment:
```suggestion
	EnableClusterBackup *bool `json:"enableClusterBackup,omitempty"`
```

Reply:
do i change it here even that it's `EnableClusterBackups` in kkp

Reply:
it's already `EnableClusterBackups` in kkp

Reply:
In that case you should change this to:

```
// EnableClusterBackups enables the Cluster Backup feature in the dashboard.
EnableClusterBackups *bool `json:"enableClusterBackups,omitempty"`
```